test/test_utils_metrics.py

```diff
@@ -0,0 +1,12 @@
+from doctr.utils import metrics
```

Comment:
Add a `import pytest` before this if you want to use the refactor I suggested below :ok_hand: